jaeger_client/config.py

```diff
@@ -81,24 +81,23 @@ class Config(object):
     _initialized = False
     _initialized_lock = threading.Lock()
 
-    def __init__(self, config, metrics=None, service_name=None):
+    def __init__(self, config, metrics_factory=None, service_name=None):
```

Comment:
this is a breaking change, how do you suggest we roll it out internally? Is it possible to accept the old `metrics` param as deprecated and construct a new factory that delegates to the old class, as well as takes care of converting tags to a single string key (similar to how expvar factory injaeger-lib works)?